src/auth/User.h

```diff
@@ -21,6 +22,8 @@
 #include "Notes.h"
 #include "sbuf/SBuf.h"
 
+#include <list>
+
```

Comment:
This new `#include` should not be needed because the new code in this file does not use `std::list` (it uses a derived type from `acl/Acl.h`).

```suggestion
```

GitHub suggestion does not show that _both_ lines should be removed (even though I applied it to both lines) -- probably a GitHub bug.